modules/cnv_uninstall_1_4.adoc

```diff
@@ -0,0 +1,49 @@
+[[uninstall_cnv]]
+=== Uninstalling {ProductName} 1.4
+
+You can uninstall {ProductName} by adding `-e apb_action=deprovision` 
```

Comment:
i would replace the 'by adding',
we have apb_action=provision when installing, 
so it should be 'by replacing the apb_action param to...'

Reply:
Oh, good catch! I'll update that shortly and then move the BZ to ON_QA.

Reply:
Nelly, how's this? https://github.com/openshift/openshift-docs/pull/13328/commits/1d86405db513624f8b4184dc6d743e85d59ea008

Reply:
/lgtm for what it's worth.